src/Mapbender/Component/CustomDoctrineParamConverter.php

```diff
@@ -28,16 +38,21 @@ public function __construct(\Doctrine\Bundle\DoctrineBundle\Registry $registry =
         $this->registry = $registry;
```

Comment:
Returning from a constructor is rarely useful. What if it is null and you return? Then the instance attribute is not set to null. But it already is null, even without the explicit default. So the check doesn't change anything. Because of the type declaration, it can only ever be null or an object anyway.

The real issue here is that the object loses most of its function without the registry. If there's a use case for that, we could do a SupportsAbsolutelyNothingParamConverter that has a working apply method but can't do anything else.